ipatests/pytest_ipa/integration/resolver.py

```diff
@@ -192,6 +203,15 @@ def _apply_state(self, state):
                 self.RESOLVED_CONF_FILE, state['resolved_config'])
         self._restart_resolved()
 
+    def uses_localhost_as_dns(self):
+        """Return true if the localhost is set as DNS server.
+
+        When systemd-resolved is in use, the DNS can be found using
+        the command resolvectldns.
+        """
+        dnsconf = self.host.run_command(['resolvectl', 'dns'])
+        return any(ip in dnsconf.stdout_text for ip in ('127.0.0.1', '::1'))
```

Comment:
Sorry, I did not notice the issue in previous review: this check will match "127.0.0.127".  
Also It will be nice to check that the address is in the "Global" section.

The output of `resolvectl dns` with IPA installed with DNS support is something like
```
# resolvectl dns
Global: 127.0.0.1
Link 2 (eth0): 10.11.5.19 10.5.30.45
```